types/stripe/index.d.ts

```diff
@@ -6449,6 +6449,91 @@ declare namespace Stripe {
              */
             percentage: number | null;
         }
+
+        interface ITaxRateCreationOptions {
+            /**
+             * The display name of the tax rate, which will be shown to users.
+             */
+            display_name: string;
+            /**
+             * This specifies if the tax rate is inclusive or exclusive.
+             */
+            inclusive: boolean;
+            /**
+             * This represents the tax rate percent out of 100.
+             */
+            percentage: number;
+            /**
+             * Flag determining whether the tax rate is active or inactive. Inactive tax rates continue to work where they are currently applied however they cannot be used for new applications.
+             */
+            active?: boolean;
+            /**
+             * An arbitrary string attached to the tax rate for your internal use only. It will not be visible to your customers.
+             */
+            description?: string;
+            /**
+             * The jurisdiction for the tax rate.
+             */
+            jurisdiction?: string;
+            /**
+             * Set of key-value pairs that you can attach to an object. This can be useful for storing additional information about the object in a structured format.
+             * Individual keys can be unset by posting an empty value to them. All keys can be unset by posting an empty value to metadata.
+             */
+            metadata?: IMetadata;
```

Comment:
`metadata` in Options interfaces should be `IOptionsMetadata` so they can accept numbers.